storage/innobase/handler/ha_innodb.cc

```diff
@@ -1243,6 +1243,16 @@ normalize_table_name_low(
 					 name to lower case */
 
 /*************************************************************//**
+Empty free list algorithm.
+Checks if buffer pool is enough to enable backoff algorithm.
+@return	true if it's not possible to enable backoff. */
+static
+bool
+innodb_empty_free_list_algorithm_check(
```

Comment:
The name should be more descriptive (what is being "check"-ed?), for example, innodb_empty_free_list_algorithm_backoff_allowed, with return condition inverted